packages/sdks-tests/src/snippet-tests/integrating-sections.spec.ts

```diff
@@ -8,15 +8,15 @@ test.describe('Integrating Sections', () => {
     await findTextInPage({ page, text: 'This is an announcement banner' });
   });
 
-  test('does not load announcement bar at `/announcements/foo`', async ({ page, packageName }) => {
-    test.skip(!['react'].includes(packageName));
+  test('does not load announcement bar at `/announcements/foo`', async ({ page }) => {
 
     await page.goto('/announcements/foo');
 
     await expect(page.locator('body')).not.toHaveText('This is an announcement banner');
   });
 
-  test('rest of the content is always present', async ({ page }) => {
+  test('rest of the content is always present', async ({ page, packageName }) => {
+    test.skip(!['sveltekit'].includes(packageName));
```

Comment:
do we need this? i think `The rest of your page goes here` text should be present in all the snippets

Reply:
Oh right. updated